test/openssl/test_bn.rb

```diff
@@ -334,6 +334,34 @@ def test_get_flags_and_set_flags
     e.set_flags(0)
     assert_equal(4, e.get_flags(OpenSSL::BN::CONSTTIME))
   end
+
+  if respond_to?(:ractor)
+    ractor
+    def test_ractor
+      return if Gem::Version.create('3.0') > Gem::Version.create(RUBY_VERSION)
+      return unless defined?(Ractor)
```

Comment:
These two lines should be unnecessary now. The `ractor` annotation seems to take care of it for us.

https://github.com/ruby/openssl/runs/4594327352?check_suite_focus=true
> ```
>  Omissions:
> 1) Omission: <OpenSSL::TestBN#test_ractor> requires Ractor [test_ractor(OpenSSL::TestBN)]
> /opt/hostedtoolcache/Ruby/2.7.5/x64/lib/ruby/gems/2.7.0/gems/test-unit-3.5.3/lib/test/unit/omission.rb:87:in `omit'
> 
> ```

Reply:
Right. Removed those lines. 7541a66